pkg/tsdb/elasticsearch/client/search_request_test.go

```diff
@@ -446,7 +449,8 @@ func TestSearchRequest(t *testing.T) {
 func TestMultiSearchRequest(t *testing.T) {
 	Convey("Test elasticsearch multi search request", t, func() {
 		Convey("Given new multi search request builder", func() {
-			b := NewMultiSearchRequestBuilder(0)
+			version2, _ := semver.NewVersion("2.0.0")
```

Comment:
I'm not sure why it was 0 tbh, I think I blindly replaced it.

looking at the test it feels like there was no reason at all (also because we never supported anything lower than 2: https://github.com/grafana/grafana/pull/33646/files/ebd3f7a33595ce6edff0fec1c0963e0e0ceffb2b#diff-95c1bcdd942efc587e6f08b4175e52e30410f94425bc990a9260fb446f131831L71 ) 

but I'll try to understand better